storage/2018-11-09/blob/accounts/models.go

```diff
@@ -0,0 +1,55 @@
+package accounts
+
+import "github.com/Azure/go-autorest/autorest"
+
+type SetServicePropertiesResult struct {
+	autorest.Response
+}
+
+type StorageServiceProperties struct {
+	// Cors - Specifies CORS rules for the Blob service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Blob service.
+	Cors *CorsRules `xml:"Cors,omitempty"`
+	// DefaultServiceVersion - DefaultServiceVersion indicates the default version to use for requests to the Blob service if an incoming request’s version is not specified. Possible values include version 2008-10-27 and all more recent versions.
+	DefaultServiceVersion string `xml:"DefaultServiceVersion,omitempty"`
```

Comment:
this'll want to be a pointer, since it's possible to omit it - based on this: https://docs.microsoft.com/en-us/rest/api/storageservices/set-blob-service-properties

> Optional. To set DefaultServiceVersion, you must call Set Blob Service Properties using version 2011-08-18 or later. DefaultServiceVersion indicates the default version to use for requests to the Blob service if an incoming request’s version is not specified. Possible values include version 2008-10-27 and all more recent versions. For more information on applicable versions, see Versioning for the Azure Storage Services.

```suggestion
	DefaultServiceVersion *string `xml:"DefaultServiceVersion,omitempty"`
```